packages/ui/src/components/atoms/BlurOverlay.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode } from 'react';
+import { styled } from 'styled-components';
+
+interface BlurOverlayProps {
+  children: ReactNode;
+}
+
+const BlurOverlayStyle = styled.div`
+  background: rgba(0, 0, 0, 0.03);
+  backdrop-filter: blur(3px);
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  left: 0px;
+  top: 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 64px;
+  gap: 10px;
+  z-index: 99;
+`;
+
+export const BlurOverlay: FC<BlurOverlayProps> = ({ children }) => (
+  <BlurOverlayStyle>{children}</BlurOverlayStyle>
+);
```

Comment:
In the `DialogBox` component we have a prop `$isModal` which makes it a popup. Prefer using and modifying that instead of creating new one.